### ISSUE TITLE
[Bug] After the table is sorted, the tooltip is misplaced

### ISSUE THREAD

**Issue by Xiaokang2022** [squidfunk/mkdocs-material#7453]
### Context

* OS: Windows 11 23H2
* Browser: Edge 127.0.2651.74
* mkdocs: `1.6.0`
* material: `9.5.31`
* Python: `3.12.0`
* tablesort: `5.3.0`

### Bug description

The way I added sorting to the table is what the documentation mentioned: https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables

When I use the following code to create a table, a bug occurs:

```markdown
| Id  | Value           |
| --- | --------------- |
| 1   | A { title="A" } |
| 2   | B               |
| 3   | C               |
| 4   | D               |
| 5   | E               |
```

Before sorting, the position of this tooltip (`"A"`) is correct:

<img width="228" alt="image" src="https://github.com/user-attachments/assets/56c366ea-c86f-4102-8f33-eafc5bd189a3">

But after sorting, this tooltip is misplaced:

<img width="199" alt="image" src="https://github.com/user-attachments/assets/7c3879cb-52c5-4cc5-9383-a68de0deee3d">

It was only when I re-scrolled the page that it came back to the correct place.

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables

### Reproduction

[9.5.31-bug.zip](https://github.com/user-attachments/files/16653556/9.5.31-bug.zip)

### Steps to reproduce

1. Create a table with a tooltip
2. Introduce [JS](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables-mkdocsyml) for table sorting
3. Sort the column that contains the tooltip
4. Check whether the tooltip is misplaced

### Browser

Edge

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Fixed in e5c9575dd. Here's what happened and how it's fixed:

https://github.com/squidfunk/mkdocs-material/blob/e5c9575dda8b4e8b689efffda2690f5925ae7919/src/templates/assets/javascripts/components/tooltip2/index.ts#L138-L148

**Comment by squidfunk** [comment]
We'll leave it open until released.
